pkg/discovery/probe/pod_probe_util_test.go

```diff
@@ -127,3 +130,39 @@ func TestBreakdownTurboPodUUID(t *testing.T) {
 		}
 	}
 }
+
+func TestIsMirrorPod(t *testing.T) {
+	table := []struct {
+		annotations map[string]string
+
+		expects bool
+	}{
+		{
+			annotations: map[string]string{
+				kubelettypes.ConfigMirrorAnnotationKey: "foo",
+			},
+			expects: true,
+		},
+		{
+			annotations: map[string]string{
+				"foo": "bar",
+			},
+			expects: false,
+		},
+		{
+			expects: false,
+		},
+	}
+	for _, item := range table {
+		pod := &api.Pod{}
+		pod.Annotations = item.annotations
+		res := isMirrorPod(pod)
+		if res != item.expects {
+			temp := ""
+			if !item.expects {
+				temp = "not"
+			}
+			t.Errorf("pod is %s a mirror pod, but isMirror returned %t", temp, res)
```

Comment:
It may be helpful to print out the pod, so when there's a failure we know right away what actually fails the test.